translations/en-us.yml

```diff
@@ -2187,9 +2187,9 @@ osf-components:
         acceptRevisionDescription: 'Update will be accepted and registration will be updated'
         rejectRevision: 'Reject update'
         rejectRevisionDescription: 'Update will be rejected and registration will remain unchanged'
-        acceptCollectionSubmission: 'Accept submission'
+        acceptCollectionSubmission: 'Accept Request'
         acceptCollectionSubmissionDescription: 'Project will appear in search results and be associated with the collection'
-        rejectCollectionSubmission: 'Reject submission'
+        rejectCollectionSubmission: 'Reject Request'
         rejectCollectionSubmissionDescription: 'Project will not appear in search results nor be associated with the collection'
         removeCollectionSubmission: 'Remove submission'
```

Comment:
I would double check with product folks to see if they want `Remove submission` changed as well. I think Blaine would be the person to ask